romanesco/plugins/swift/executor.py

```diff
@@ -0,0 +1,112 @@
+import os
+import re
+import select
+import subprocess
+import sys
+import tempfile
+
```

Comment:
@zachmullen I essentially lifted code from this file verbatim from your `docker` plugin. Should there be a (possibly core) shell execution mode where `docker` and `swift` (among others) utilize components of it to customize it?